cmd/kubeless/list_test.go

```diff
@@ -129,4 +160,8 @@ func TestList(t *testing.T) {
 	// yaml output
 	output = listOutput(t, client, "myns", "yaml", []string{})
 	t.Log("output is", output)
+
+	// wide output
+	output = listOutput(t, client, "myns", "wide", []string{})
+	t.Log("output is", output)
```

Comment:
Do some basic checks to assert the presence of various expected strings in the output of each of these, otherwise you aren't going to know if it's broken without actually looking at the test output.